src/main/java/de/oelrichsgarcia/studipTelegramBot/studipTelegramBot/studip/StudIPBot.java

```diff
@@ -257,12 +264,18 @@ public void downloadFilesAndCreateFolders(ArrayList<StudIPObject> objects, Path
                     if (object.getUpdated().getTime() > file.lastModified()) {
                         Sout("Updating outdated file " + basePath + "/" + object.getName() + " " + fileSize);
                         downloadFile((StudIPFile) object, basePath);
+                        if(useGoogleDrive) {
+                            uploadToGoogleDrive((StudIPFile) object, basePath, drive_root_folder);
+                        }
                         updateSummary.addUpdatedFile((StudIPFile) object);
 
                     }
                 } else {
                     Sout("Downloading new file " + basePath + "/" + object.getName() + " " + fileSize);
                     downloadFile((StudIPFile) object, basePath);
+                    if(useGoogleDrive) {
```

Comment:
Please load the GoogleDriveDownloader during config loading so the StandardDownloader gets replaced by yours